CODE_QUALITY.md

```diff
@@ -1,9 +1,14 @@
 ### Code Quality Requirements for Homeworks
 
-The code submitted to this repository is required to be formatted with [Prettier](https://github.com/prettier/prettier) (for config details see "prettier" section in `package.json`) and match [Airbnb code style](https://github.com/airbnb/javascript). Please follow these steps:
+The code submitted to this repository is required to be formatted with [Prettier](https://github.com/prettier/prettier) (for config details see "prettier" section in `package.json`) and match [Airbnb code style](https://github.com/airbnb/javascript).
+
+Install and use [yarn](https://yarnpkg.com/en/docs/install)
+instead of `npm`.
+
+Please follow these steps:
 1. After cloning this repository install the dependencies (run `yarn` in project directory). This will install the following tools:
     - eslint
-    - eslint-config-airnbn-base
+    - eslint-config-airbnb-base
```

Comment:
Propose to use one of:
https://standardjs.com
https://github.com/google/eslint-config-google

Each of these rules looks much better on the backend and was created especially for this purpose, instead of airbnb.